CardDictionaries/ArcaneRising/ARCWizard.php

```diff
@@ -305,6 +310,11 @@ function CurrentEffectArcaneModifier($source, $player)
           $modifier += $effectArr[1];
           $remove = true;
           break;
+        case "ROS186": case "ROS187": case "ROS188":
+          if($currentTurnEffects[$i+1] != $player) break;
+          $modifier += $effectArr[1];
+          $remove = true;
+          break;
```

Comment:
same as my previous comment but with idea that this should come after ROS033 which is currently below it.